cmd/nitrite/main.go

```diff
@@ -5,53 +5,52 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
+	"log/slog"
 	"os"
 
 	"github.com/blocky/nitrite"
 )
 
 var (
-	fDocument = flag.String("attestation", "", "Attestation document in standard Base64 encoding")
+	fAttestation = flag.String("attestation", "", "Attestation document in standard Base64 encoding")
+	fAllowDebug  = flag.Bool("allowdebug", false, "Allow verification of attestation generated in debug mode")
 )
 
 func main() {
 	flag.Parse()
 
-	if "" == *fDocument {
+	if "" == *fAttestation {
 		flag.PrintDefaults()
 		os.Exit(1)
 	}
```

Comment:
I find using the `-attestation` flag clunky because I would much rather cat in a file containing my attestation string than copy and paste as a flag. Could we either (1) remove the attestation flag and always read from stdin or (2) add the option to pass the string via stdin (shown below)
```golang
	if *fAttestation == "" {
		flag.PrintDefaults()
		os.Exit(1)
	}

    // This allows a user to pipe in an attestation string from other commands. For example:
    //     cat ./testdata/nitro_attestation.b64 | go run ./cmd/nitrite/ -attestation -
	if *fAttestation == "-" {
		bytes, err := io.ReadAll(os.Stdin)
		if err != nil {
			err = fmt.Errorf("reading base64 attestation string from stdin: %w", err)
			slog.Error(err.Error())
			os.Exit(2)
		}
		*fAttestation = string(bytes)
	}
```

Reply:
done